adapters/buenos-aires.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+import { default as moment } from 'moment-timezone';
+import cheerio from 'cheerio';
+import async from 'async';
+const request = baseRequest.defaults({timeout: REQUEST_TIMEOUT});
+
+export const name = 'buenos aires';
+export function fetchData (source, callback) {
+  // list of requests used to get data
+  const tasks = generateTasks(source);
+
+  async.parallel(tasks, (err, response) => {
+    if (err) {
+      return callback(new Error('Failure to load data urls.'));
+    }
+
+    // wrap in try catch to handle possible errors
+    try {
+      const data = formatData(response);
+      if (data === undefined) {
+        return callback(new Error('Failure to parse data.'));
+      }
+      callback(null, data);
+    } catch (e) {
+      return callback(new Error('Unknown adapter error.'));
+    }
+  });
+}
+
+// makes a requests list of requests for each station to each pollutant
+const generateTasks = (source) => {
+  const linkRequests = [];
+  ['CENTENARIO', 'CORDOBA', 'LA BOCA'].forEach((station) => {
+    ['CO', 'NO2', 'PM10'].forEach((pollutant) => {
+      const makeLinks = (station, pollutant) => {
+        const day = moment().date().toString();
```

Comment:
I think `moment().format()` is better suited here.

```js
>> var d = moment();
>> d.format("D");
"15"
>> d.format("MM");
"07"
>> d.format("Y")
"2017"
```

Also, is `moment.tz` useful here?